robotcontainer.py

```diff
@@ -132,7 +132,8 @@ def configureButtonBindings(self):
         )
 
         ModifiableJoystickButton(self.operatorInterface.alignClosestWaypoint).whileTrue(
```

Comment:
change this from alignClosestWaypoint to just AlignAndAim to be consistent. Also, why does it only aim when a button is held? imo should be whenever we have a note and a tag is in vision (maybe with some way to disable for trap?)

Reply:
mistake replacement